aiohttp/test_utils.py

```diff
@@ -537,8 +537,11 @@ def make_mocked_request(
     """
     task = mock.Mock()
     if loop is ...:
-        loop = mock.Mock()
-        loop.create_future.return_value = ()
+        try:
+            loop = asyncio.get_running_loop()
+        except RuntimeError:
```

Comment:
Could you explain this? A code comment would be useful.

Reply:
Frankly these uses of the loop parameter should probably deprecated and removed.

Reply:
added a comment in 9fd76882de2a9eefd8d4284671dd3556694a9d36

Reply:
@bdraco does this mean that `asyncio.get_running_loop()` raises a `RuntimeError` when a no loop not running? That's what I wanted to understand initially.

Reply:
Yes it does

Reply:
Marking as resolved.